core/src/com/mygdx/game/models/Command.java

```diff
@@ -5,47 +5,50 @@
  */
 public class Command {
 
-    /* Type of action the command represents */
+    /** Type of action the command represents */
     private String actionType;
 
-    /* Acting unit's ID */
+    /** Acting unit's ID */
     private int unitId;
 
-    /* Target unit's ID. Not necessary, commented out for now. */
-//     private int targetUnitId;
-
-    /* Targeted position */
+    /** The start and targeted positions of the action. */
+    private Position startPosition;
     private Position targetPosition;
 
-    public Command(String actionType, int unitId /*, int targetUnitId */, Position targetPosition) {
+    /** The distance between the start and end positions. */
+    private int distance;
+
+    /** Whether there is line of sight between the start and end positions */
+    private boolean hasLineOfSight;
```

Comment:
Why is this in this class. It should be impossible to create an action without line of sight, or line of sight doesn't matter.

Reply:
has line of sight should be a function in hexmap